tests/dao_tests/test_dv_order_dao.py

```diff
@@ -78,8 +77,6 @@ def test_insert_biobank_order(self):
         self.assertEqual(put_response["version"], 4)
         self.assertEqual(put_response["meta"]["versionId"].strip("W/"), '"4"')
         self.assertEqual(put_response["barcode"], "SABR90160121INA")
-        # self.assertEqual(put_response["biobankOrderId"], "12345")
-        self.assertEqual(put_response["biobankStatus"], "Delivered")
         self.assertEqual(put_response["order_id"], 999999)
 
```

Comment:
You should add an assert from a GET request showing it is inserted and fetchable from the DB.